generic3g/FieldDictionary.F90

```diff
@@ -229,11 +223,24 @@ function get_standard_name(this, alias, rc) result(standard_name)
       _RETURN(_SUCCESS)
    end function get_standard_name
 
+   function get_regrid_method(this, standard_name, rc) result(regrid_method)
+      class(FieldDictionary), target, intent(in) :: this
+      character(*), intent(in) :: standard_name
+      integer, optional, intent(out) :: rc
+      type(ESMF_RegridMethod_Flag), allocatable :: regrid_method ! result
+
+      type(FieldDictionaryItem), pointer :: item
+      integer :: status
+
+      item => this%entries%at(standard_name, _RC)
```

Comment:
If this line fails, then no value is assigned for the function.

Reply:
This seems to be the way all other accessors work in `MAPL_FieldDictionary`. Since we're going to replace this with `NUOPC_FieldDictionary`, I'm letting this one be.